DotnetFoundation/DotnetFoundation.Api/Controllers/AuthenticationController.cs

```diff
@@ -45,46 +61,101 @@ public async Task<ActionResult<BaseResponse<AuthenticationResponse>>> RegisterAs
     public async Task<ActionResult<BaseResponse<AuthenticationResponse>>> LoginAsync(LoginRequest request)
     {
         BaseResponse<AuthenticationResponse> response = new(ResponseStatus.Fail);
+        try
+        {
+            response.Data = await _authenticationService.LoginAsync(request).ConfigureAwait(false);
+            response.Status = ResponseStatus.Success;
 
-        response.Data = await _authenticationService.LoginAsync(request).ConfigureAwait(false);
-        response.Status = ResponseStatus.Success;
+            return Ok(response);
+        }
+        catch (InvalidCredentialsException ex)
+        {
+            response.Message = ex.Message;
 
-        return Ok(response);
+            response.Status = ResponseStatus.Error;
+            return BadRequest(response);
+        }
+        catch (LockoutException ex)
```

Comment:
You don't need to create exception here. As per discussion, there is login and logout. However, when logging in, you need to ensure that DeleteAccountAt is becoming null during successful login.

Reply:
removed